### PR TITLE
[15.0][FIX] project_mis_report_budget: new recalculate margin and wrong search

### PR DESCRIPTION
**Pull request by ChristianSantamaria**
Recalculation of margin on a monthly basis to balance budgeted and actual revenues

### CODE REVIEW

File: project_mis_report_budget/models/project_project.py [hunk 1]
```diff
@@ -54,41 +54,54 @@ def write(self, vals):
...
+                            record.last_margin = -1
+                        else:
+                            record.last_margin = ((incomes - month_income) - all_expenses) / (incomes - month_income)
 
-                    budget_item_id = budget_ids.item_ids.search(['&', ('kpi_expression_id.kpi_id.kpi_type', '=', 'margin'), ('date_to', '>=', record.last_close_date)])
-                    budget_item_id.sudo().write({
-                        'amount': new_margin
-                    })
+                        new_margin = round(100 * record.last_margin, 2)
+                        budget_item_id = budget_ids.item_ids.search(['&', ('kpi_expression_id.kpi_id.kpi_type', '=', 'margin'), ('date_to', '>=', record.last_close_date)])
```
Comment (dalonsod) [inline project_mis_report_budget/models/project_project.py]: Why is this a `search` instead of `filtered`?

**Review by dalonsod — CHANGES_REQUESTED** [review]
See comment. And also review commit message, because current one does not refer to bug that is fixed by this PR

File: project_mis_report_budget/models/project_project.py (hunk 1 above)
Comment (ChristianSantamaria) [reply project_mis_report_budget/models/project_project.py]: My fault, old code

**Review by dalonsod — APPROVED** [review]
Only code review :+1:

**Pull request merged**